src/sections/dieFounder.html

```diff
@@ -30,31 +30,35 @@ <h3 class="mt-12 mb-6 text-white font-railwayBold">
           dem Norden und Süden anbieten.
         </p>
       </div>
-      <div class="relative z-10 ml-80 mr-96">
+      <div class="relative lg:z-10 lg:ml-80 lg:mr-96 ml-6 mr-6">
         <img src="../../images/Mathias_Team.webp" alt="" />
       </div>
     </section>
     <section
-      class="relative flex flex-row text-white bg-no-repeat bg-cover -mt-[220px] h-[400px] z-0 justify-between"
+      class="lg:relative flex flex-row text-white lg:bg-no-repeat lg:bg-cover lg:-mt-[300px] lg:h-[550px] z-0 justify-between gap-4 -mt-36 md:h-[300px] md:-mt-40"
       style="background-image: url(../../images/bg_hero_main.webp)"
     >
-      <div class="mt-72 ml-72">
-        <h6 class="mb-6 text-xs font-railwayBold">Volker Maiborn:</h6>
+      <div class="lg:mt-96 lg:pl-96 mx-4 mt-36">
+        <h6 class="lg:mb-6 mb-4 text-xs lg:font-railwayBold">
+          Volker Maiborn:
+        </h6>
         <p class="text-xs font-railwayItalic">
           “d diam nonumy eirmod <br />
           tempor invidunt ut labore et <br />
           dolore magna aliquyam erat, sed diam“
         </p>
       </div>
-      <div class="mt-20 pt-20 mr-24">
-        <h6 class="mb-6 text-xs font-railwayBold">Marie ohnesorge</h6>
+      <div class="lg:mt-20 lg:pt-24 lg:pr-48 mt-32 xl:mt-40">
+        <h6 class="lg:mb-6 mb-2 text-xs font-railwayBold -ml-2">
+          Marie ohnesorge
+        </h6>
         <p class="text-xs font-railwayItalic">
           “d diam nonumy eirmod <br />
           tempor invidunt ut labore et <br />dolore magna aliquyam erat, sed
           diam“
         </p>
       </div>
-      <div class="mt-28 mr-52 pt-28">
+      <div class="lg:mt-48 lg:pr-[400px] lg:pt-28 mt-36">
```

Comment:
And on tablet maybe you can try to make it responsive so that the name of "Hugo Obi" can still be available.

![image](https://github.com/divinecharlotte/north-star-team7/assets/81089765/7d17753c-8643-4102-b44c-63e11a9cfdc1)